tensorflow/contrib/learn/python/learn/preprocessing/text.py

```diff
@@ -155,38 +155,45 @@ def fit(self, raw_documents, unused_y=None):
     self.vocabulary_.freeze()
     return self
 
-  def fit_transform(self, raw_documents, unused_y=None):
+  def fit_transform(self, raw_documents, unused_y=None, return_docs_len=False):
     """Learn the vocabulary dictionary and return indexies of words.
 
     Args:
       raw_documents: An iterable which yield either str or unicode.
       unused_y: to match fit_transform signature of estimators.
+      return_docs_len: Boolean if True yields number of tokens also.
 
     Returns:
       x: iterable, [n_samples, max_document_length]. Word-id matrix.
+        Additional token count also returned is `return_docs_len= True`.
```

Comment:
add backticks around pseudocode in brackets. Make the second sentence: "If `return_doc_len` is `True`, returns a tuple `(x, n_tokens)`, where x is the word-id matrix as before, and `n_tokens` is the number of tokens in the document."